docs/getting-started/quickstart.mdx

```diff
@@ -71,7 +68,7 @@ you'll be redirected to your CLI session and can proceed with creating your supe
 <Step
 id="supergraph-init"
 language="bash"
-code={`ddn supergraph init mysupergraph\ncd mysupergraph`}
+code={['ddn supergraph init mysupergraph', 'cd mysupergraph']}
```

Comment:
I understand why we're breaking up these and other code blocks, but I wonder if we're sacrificing speed for a lower cognitive load here. Do we also run the risk of a user running — for example — the `init` command and then not moving into the correct directory?

Reply:
👍